capabilities/go.mod

```diff
@@ -0,0 +1,84 @@
+module github.com/smartcontractkit/chainlink-framework/capabilities
+
+go 1.24.0
+
+require (
+	github.com/ethereum/go-ethereum v1.14.11
```

Comment:
Why is geth showing up in this module?

Reply:
Because it uses abi encoding in for the reports https://github.com/smartcontractkit/chainlink-framework/blob/3cdd888aff2090bc14a83e09070280d2eb2bb6e3/capabilities/writetarget/beholder/report/datafeeds/types.go#L9-L10

Reply:
But this repo is meant to be chain-agnostic. Why are EVM specifics showing up here?

Reply:
Both Aptos and EVM use the ABI encoding/decoding here, so atm all chains that implement the generalized write target need to abi encode/decode. We do have a ticket in the future to further generalize this though. https://smartcontract-it.atlassian.net/browse/BCFR-1221

Reply:
Those sound like separate concerns to me. What advantage is there to putting code here, which is known to be tech debt, as opposed to putting it somewhere EVM specific?

Reply:
Where do you suggest we put this code? I thought this was a fine place as it avoids import cycles

Reply:
That might depend on how we expect it to evolve, but it seems like mixing layers to me. Why is this repo more appealing than e.g. https://github.com/smartcontractkit/chainlink-evm?

Reply:
One argument is that Aptos is not an EVM chain, so it shouldn't use a component / framework that is housed in an EVM repo.

Reply:
OK, but we can't have it both ways. What is our term for things which act like EVM but which are not _the_  EVM?

Reply:
Not sure if there is a term for that? And afaik we don't have a repo for components that fall into that classification so I thought `chainlink-framework` was an ok landing spot. But if you strongly feel like `chainlink-evm` is better, I don't mind refactoring it.

Reply:
I don't have a strong preference, but we need to be doing something consistent in any case, and it's not obvious to me how we arrived here or where we intend to go.

Why does using this repo, and a chain agnostic name, seem less wrong than having `chainlink-aptos` import the same module from `chainlink-evm/capabilities`?

What if this module was simply named `evm-capabilities`?

Reply:
If they are equally wrong, then IMHO we should chose the one with the smaller blast radius.

What do you mean about separation? I am not advocating for separating unrelated things. However, chain agnostic and chain specific things are separate by design.

> Until we have a place for evm-like code, this seems to be the best place for this.

What would that look like? What is the blocker to doing that now?
Regardless, I think this just enforces the idea that chainlink-evm should be used. If we are transitioning to something better, then we definitely don't need to be polluting this repo in the mean time.

Reply:
Ok, lets go with `chainlink-evm` then. Again, I don't have a preference my main priority is getting this code unblocked ASAP since we are aiming for the release cut tomorrow.

Reply:
Here is the [chainlink-evm](https://github.com/smartcontractkit/chainlink-evm/pull/2) PR, refactoring this PR now.